packages/appstore/src/components/banners/book-banner/book-banner.scss

```diff
@@ -0,0 +1,46 @@
+.book-banner-template {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1.6rem;
+    margin-block-end: 2.4rem;
+    border-radius: 8px;
+    background: rgba(0, 136, 50, 0.08);
```

Comment:
finalising by UI/UX team

Reply:
@NikitK-deriv isn't this color available in the theme.scss?

Reply:
as I wrote, color is finalising

Reply:
color is finalised and updated. actually accepted